platform/mds/mds-web/src/main/resources/webapp/js/directives.js

```diff
@@ -3213,4 +3213,23 @@
             }
         };
     });
+
+    directives.directive('embeddedMdsGrid', function($http, $templateCache, $compile) {
```

Comment:
This is not really an embedded mds grid. Its the layout of the page, that might contain the embedded MDS grid if the I happen to choose the right tab. Rename this to something along the lines of TabLayoutWithMdsGrid or something.